packaging_automation/citus_package.py

```diff
@@ -162,30 +163,28 @@ def sign_packages(base_output_path: str, sub_folder: str, secret_key: str, passp
 def get_postgres_versions(os_name: str, input_files_dir: str) -> Tuple[List[str], List[str]]:
     release_versions = []
     nightly_versions = []
+
     if platform_postgres_version_source[os_name] == PostgresVersionDockerImageType.single:
         release_versions = ["all"]
         nightly_versions = ["all"]
     else:
-        with open(f"{input_files_dir}/pkgvars", "r") as reader:
-            content = reader.read()
-            lines = content.splitlines()
-            for line in lines:
-                if line.startswith("releasepg"):
-                    release_version_assignment = line
-                if line.startswith("nightlypg"):
-                    nightly_version_assignment = line
-            if release_version_assignment is None or "=" not in release_version_assignment or len(
-                    release_version_assignment.split("=")) != 2:
-                raise ValueError(
-                    f"Release version in pkglatest is not well formatted. Expected format: releasepg=12,13 "
-                    f"Actual Format:{release_version_assignment}")
-            if nightly_version_assignment is None or "=" not in nightly_version_assignment or len(
-                    nightly_version_assignment.split("=")) != 2:
+        pkgvars_config = dotenv_values(f"{input_files_dir}/{PKGVARS_FILE}")
+        release_versions_str = pkgvars_config['releasepg']
+        if ',' not in release_versions_str:
```

Comment:
I do not like this conditional here. Does that mean we can not actually create a package for a single PG version?

We used to do that when we need to create a single local package for testing purposes for example.

I think `releasepg=12` is a well formatted line for `pkgvars` file. What do you think?

Reply:
Removed ',' validation